Library/Homebrew/rubocops/extend/formula_cop.rb

```diff
@@ -0,0 +1,67 @@
+module RuboCop
+  module Cop
+    module Homebrew
+      class FormulaCop < Cop
+        @registry = Cop.registry
+
+        def on_class(node)
+          # This method is called by RuboCop and is the main entry point
+          class_node, parent_class_node, body = *node
+          return unless a_formula_class?(parent_class_node)
+          audit_formula(node, class_node, parent_class_node, body)
```

Comment:
Add a `return unless respond_to?(:audit_formula)` here, so you can remove `Homebrew/FormulaCop` from the RuboCop config.